extensions/amp-story/1.0/bookend/components/article.js

```diff
@@ -77,6 +78,7 @@ export class ArticleComponent {
       domainName,
       type: articleJson['type'],
       title: articleJson['title'],
+      alt: articleJson['alt'],
```

Comment:
This could be called `image-alt-text` since it's only applied to images.

Reply:
I tried renaming and ran into an issue with the following test:

https://github.com/ampproject/amphtml/blob/1b7080708cb0260689e24c8531d29354030ba700/extensions/amp-story/1.0/test/test-amp-story-bookend.js#L464-L500

I isolated the value of alt between `config` and `userJson` and see that the following passes: 

```js
expect(config.components[1].alt).to.equal(userJson.components[1]['image-alt-text']);

// expect(config.components[1]).to.deep.equal(userJson.components[1]);
```

I think `alt` vs `image-alt-text` might be causing the issue.  Do you have any thoughts on how to preserve the original test integrity with the rename?

Reply:
Thanks for looking into this @nickFalcone
I looked into this a little deeper and `alt` is the naming convention used in our codebase. 
It's safe to stick with `alt` :)

Reply:
Thanks @processprocess, sticking with `alt` all tests pass (including new separated tests) 😄